distributed/utils_test.py

```diff
@@ -1051,7 +1068,7 @@ async def test_foo(scheduler, worker1, worker2, pytest_fixture_a, pytest_fixture
     worker_kwargs = merge(
         dict(
             memory_limit=system.MEMORY_LIMIT,
-            death_timeout=15,
+            death_timeout=min(15, int(watch.leftover())),
```

Comment:
Isn't this bound too early? `watch.start()` was only called two `dict()` calls ago, so this will always be `min(15, 60)`

Reply:
Ah this is for when someone passes `@gen_cluster(timeout=14)` (or lower)